incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -197,6 +201,56 @@ func TestNonRBACTypes(t *testing.T) {
 	}
 }
 
+func TestPropagateConflict(t *testing.T) {
+	// Create a forest with "a" as the parent and "b" and "c" as the children.
+	f := foresttest.Create("-aa")
+
+	tests := []struct {
+		name         string
+		inNamespaces []string
```

Comment:
Since each namespace name is a single character, this can just be a string (that was on purpose 🙂 )

Although with that said - when we start testing exceptions, this code could become complicated enough that we'll need maps/lists anyway.